code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -37,20 +44,46 @@
 	var/open = FALSE
 	var/max_n_of_items = 10
 	var/efficiency = 0
+	/// If we use a cell instead of powernet
+	var/cell_powered = FALSE
+	/// The cell we charge with
+	var/obj/item/stock_parts/cell/cell
+	/// The cell we're charging
+	var/obj/item/stock_parts/cell/vampire_cell
+	/// Capable of vampire charging PDAs
+	var/vampire_charging_capable = FALSE
+	var/vampire_charging_enabled = FALSE
```

Comment:
documentation. I was about to confuse one with the other too.

Reply:
Ah yes, of all the vars that should have been doc'd, I missed the important ones